src/flow-spec/IResource.ts

```diff
@@ -5,6 +5,10 @@ export interface IResourceValue {
   content_type: string
   mime_type?: string
   modes: SupportedMode[]
+  /**
+   * Value will accept alphanumerics, white spaces, file path, expressions, conditions,
+   * @pattern ^[\w \\\/@:,.+*^<>=()"'-]+$
```

Comment:
I'm curious, how is the `@pattern …` comment used? It’s not among the jsdoc tags

Reply:
it will transformed into jsonSchem `@pattern` https://json-schema.org/understanding-json-schema/reference/regular_expressions.html when we generate them

Reply:
Thanks a lot, Safydy!
I thought the json files were manually edited, didn't know they were autogenerated.